sndpeek: init at 1.4

laikq opened the pull request:
<!-- Nixpkgs has a lot of new incoming Pull Requests, but not enough people to review this constant stream. Even if you aren't a committer, we would appreciate reviews of other PRs, especially simple ones like package updates. Just testing the relevant package/service and leaving a comment saying what you tested, how you tested it and whether it worked would be great. List of open PRs: <https://github.com/NixOS/nixpkgs/pulls>, for more about reviewing contributions: <https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#sec-reviewing-contributions>. Reviewing isn't mandatory, but it would help out a lot and reduce the average time-to-merge for all of us. Thanks a lot if you do! -->
###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

laikq committed:
maintainers: add laikq

mmahut commented on maintainers/maintainer-list.nix:
```diff
@@ -2569,6 +2569,11 @@
     github = "kylewlacy";
     name = "Kyle Lacy";
   };
+  laikq = {
```
Please split this into a separate commit.

mmahut commented:
Init packages should always go to master first.

laikq committed:
sndpeek: init at 1.4

This version compiles with ALSA, although other modes of compilation could be
imagined.

laikq commented:
Ahoy, I hope this does the trick -- I'm very much a newbie, this is my first pull request to a larger project and I hope I can contribute something and learn more about how this project works.

Lassulus commented on pkgs/applications/audio/sndpeek/default.nix:
```diff
@@ -0,0 +1,56 @@
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://soundlab.cs.princeton.edu/software/sndpeek/files/sndpeek-1.4.tgz";
```
```suggestion
    url = "https://soundlab.cs.princeton.edu/software/sndpeek/files/sndpeek-${version}.tgz";
```

laikq replied:
Ah, yes :) Should I force push this change, or add a commit?

Lassulus replied:
force push

Lassulus reviewed (approved):
tested with nix-review, seems to work
